src/main.go

```diff
@@ -58,6 +50,19 @@ func main() {
 		}
 	}
 
+	// Update mechanism
+	update := make(chan struct{}, 1)
+	if len(args) > 0 && args[0] == "version" {
```

Comment:
If the command was `--version`, then ...? I don't understand what `update <- struct{}{}`, so I can't quite figure out what difference this check makes.